archetypes/author.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "" # Name
+image: ""
+email: ""
+social:
+  - icon : "ti-linkedin" # themify icon pack : https://themify.me/themify-icons
+    link : "#" # Linkedin
+  - icon : "ti-twitter-alt" # themify icon pack : https://themify.me/themify-icons
+    link : "#" # Twitter
+  - icon : "ti-github" # themify icon pack : https://themify.me/themify-icons
+    link : "#" # Github
+---
+
+<!--Write your function below-->
```

Comment:
This confused me a bit, could we change this to `<!-- Write your job title or function below, should be one line -->`